src/utilities/gql.ts

```diff
@@ -1,10 +1,188 @@
+  DELETE_SCHEDULING_CONDITION_TAGS = 'delete_scheduling_condition_tags',
+  DELETE_SCHEDULING_GOAL_DEFINITION_TAGS = 'delete_scheduling_goal_definition_tags',
+  DELETE_SCHEDULING_GOAL_METADATA = 'delete_scheduling_goal_metadata_by_pk',
+  DELETE_SCHEDULING_GOAL_METADATA_TAGS = 'delete_scheduling_goal_tags',
+  DELETE_SCHEDULING_GOAL_MODEL_SPECIFICATIONS = 'delete_scheduling_model_specification_goals',
+  DELETE_SCHEDULING_GOAL_TAGS = 'delete_scheduling_goal_tags',
+  DELETE_SCHEDULING_SPECIFICATION = 'delete_scheduling_specification',
+  DELETE_SCHEDULING_SPECIFICATION_CONDITIONS = 'delete_scheduling_specification_conditions',
+  DELETE_SCHEDULING_SPECIFICATION_GOALS = 'delete_scheduling_specification_goals',
+  DELETE_SEQUENCE = 'delete_sequence_by_pk',
+  DELETE_SEQUENCE_TO_SIMULATED_ACTIVITY = 'delete_sequence_to_simulated_activity_by_pk',
+  DELETE_SIMULATION_TEMPLATE = 'delete_simulation_template_by_pk',
+  DELETE_TAG = 'delete_tags_by_pk',
+  DELETE_USER_SEQUENCE = 'delete_user_sequence_by_pk',
+  DELETE_VIEW = 'delete_view_by_pk',
+  DELETE_VIEWS = 'delete_view',
+  DENY_MERGE = 'deny_merge',
+  DUPLICATE_PLAN = 'duplicate_plan',
+  EXPAND_ALL_ACTIVITIES = 'expandAllActivities',
+  EXPANSION_RULE = 'expansion_rule_by_pk',
+  EXPANSION_RULES = 'expansion_rule',
+  EXPANSION_RULE_TAGS = 'expansion_rule_tags',
+  EXPANSION_RUNS = 'expansion_run',
+  EXPANSION_SETS = 'expansion_set',
+  EXTENSIONS = 'extensions',
+  GET_ACTIVITY_EFFECTIVE_ARGUMENTS = 'getActivityEffectiveArguments',
+  GET_ACTIVITY_TYPE_SCRIPT = 'getActivityTypeScript',
+  GET_COMMAND_TYPE_SCRIPT = 'getCommandTypeScript',
+  GET_CONFLICTING_ACTIVITIES = 'get_conflicting_activities',
+  GET_EDSL_FOR_SEQ_JSON = 'getEdslForSeqJson',
+  GET_MODEL_EFFECTIVE_ARGUMENTS = 'getModelEffectiveArguments',
+  GET_NON_CONFLICTING_ACTIVITIES = 'get_non_conflicting_activities',
+  GET_SEQUENCE_SEQ_JSON = 'getSequenceSeqJson',
+  GET_USER_SEQUENCE_SEQ_JSON = 'getUserSequenceSeqJson',
+  INSERT_ACTIVITY_DIRECTIVE = 'insert_activity_directive_one',
+  INSERT_ACTIVITY_DIRECTIVE_TAGS = 'insert_activity_directive_tags',
+  INSERT_ACTIVITY_PRESET = 'insert_activity_presets_one',
+  INSERT_CONSTRAINT_DEFINITION = 'insert_constraint_definition_one',
+  INSERT_CONSTRAINT_DEFINITION_TAGS = 'insert_constraint_definition_tags',
+  INSERT_CONSTRAINT_METADATA = 'insert_constraint_metadata_one',
+  INSERT_CONSTRAINT_MODEL_SPECIFICATION = 'insert_constraint_model_specification_one',
+  INSERT_CONSTRAINT_SPECIFICATIONS = 'insert_constraint_specification',
+  INSERT_CONSTRAINT_TAGS = 'insert_constraint_tags',
+  INSERT_EXPANSION_RULE = 'insert_expansion_rule_one',
+  INSERT_EXPANSION_RULE_TAGS = 'insert_expansion_rule_tags',
+  INSERT_MISSION_MODEL = 'insert_mission_model_one',
+  INSERT_PLAN = 'insert_plan_one',
+  INSERT_PLAN_SNAPSHOT_TAGS = 'insert_plan_snapshot_tags',
+  INSERT_PLAN_TAGS = 'insert_plan_tags',
```

Comment:
Seeing a regression against develop where a user who does not have edit permission on a plan is able to edit the plan tags in the Plan Metadata panel though only tag creation works and the resulting association to the plan fails.